views/dashboard-application.hbs

```diff
@@ -24,6 +24,27 @@
     <script src='/js/dashboard-application.js'></script>
 </head>
 <body>
+
+    {{#if user.hasApplication}}
+
+    <!-- NAVBAR -->
+    <div class='wrapper'>
```

Comment:
You don't have to wrap duplicated code like the navbar and the div wrapper with the if statements. If you carefully place the if statements, you should not have to duplicate that code.